example/App.svelte

```diff
@@ -0,0 +1,13 @@
+<script>
+  import { NotificationDisplay, notifier } from '../src'
```

Comment:
I wasn't able to get the demo app to run. Is it worth adding a readme to tell people how to run it?